sdk/storage/ci.yml

```diff
@@ -22,6 +22,7 @@ pr:
     - sdk/storage/
     - sdk/resourcemanager/
     - sdk/compute/
+    - sdk/keyvault/
```

Comment:
How does adding this extra trigger path fix that associated issue?

Reply:
I don't think it does.. I think I'm adding it in the wrong place. Trying to figure out the correct parameter to use.

Reply:
I need to specify additional Paths for the sparse checkout - something like:
```
  template: ../../eng/common/pipelines/templates/steps/sparse-checkout.yml
  parameters:
    Paths:
      include:
      - sdk/storage/
      - sdk/keyvault/
```

Reply:
For sparse checkout we include everything except the test recordings. See https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/ci.tests.yml#L44

Even looking at the logs I see that we are building the keyvault library successfully in the test job.

Reply:
See https://dev.azure.com/azure-sdk/public/_build/results?buildId=1619591&view=logs&j=45f79b3a-2624-51cc-bc09-5c8a7cc7b4c9&t=a5b29a05-b133-5d99-2802-d1ab9b116429&l=36 

If keyvault directory didn't exist that would fail to restore/build.

Reply:
Agreed, but when I turn off sparse checkout the failure goes away, so it seems like it is the cause.

Reply:
> For sparse checkout we include everything except the test recordings. See https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/ci.tests.yml#L44

I suspect that the sparse settings that are used for the dynamic test generation are somehow overriding the settings that are used for the test runs.

Reply:
This was a false alarm. It was unrelated to sparse checkout.